bebraven/platform
# Update LTI launch URL for Fellows viewing attendance events

### Description

**Task**: https://app.asana.com/0/1174274412967132/1197571375872706

This PR changes the LTI launch URL for fellows viewing attendance assignments in the Accelerator course. It's prep work for the actual code that lets LCs take attendance for learning labs and 1-1s, etc.

### Motivation and Context

The LTI launch URL for attendance event assignments that Fellows see used to be `AttendanceEventSubmission#launch`.

This PR changes it to `AttendanceEventSubmissionAnswer#launch`. There are two reasons for this:
1. Our data model has an `Answer` per `for_user`, where the `for_user` is a fellow the LC is taking attendance for. This lets us be as restrictive as possible with our policy, because we can check whether `current_user == answer.for_user`.
2. This frees up `AttendanceEventSubmission#launch` to use for the **Take Attendance** link in the LC Playbook course.

The second part will be implemented in follow-up PRs.

### Test

- rspec
- Add an attendance event from `course#edit` page, verify it still renders in Canvas:
<img width="1792" alt="Screen Shot 2021-01-20 at 1 02 49 PM" src="https://user-images.githubusercontent.com/62911934/105234236-cdcfe600-5b1f-11eb-9e59-226064668561.png">
- Check the assignment settings for updated LTI launch URL:
<img width="1792" alt="Screen Shot 2021-01-20 at 1 03 27 PM" src="https://user-images.githubusercontent.com/62911934/105234352-fbb52a80-5b1f-11eb-883a-c9029946151b.png">

### Types of changes




### Checklist:
